test-data/unit/check-functions.test

```diff
@@ -3007,3 +3007,75 @@ class C(A):
     def f(self, y: int | str) -> str: pass
 [typing fixtures/typing-full.pyi]
 [builtins fixtures/tuple.pyi]
+
+[case requireExplicitOverrideMethod]
+# flags: --strict-override-decorator --python-version 3.12
+from typing import override
+
+class A:
+    def f(self, x: int) -> str: pass
+
+class B(A):
+    @override
+    def f(self, y: int) -> str: pass
+
+class C(A):
+    def f(self, y: int) -> str: pass  # E: Method "f" is not marked as override but is overriding a method in a base class
+[typing fixtures/typing-override.pyi]
```

Comment:
Should this new fixture also be used for the existing override tests above? (Which I wrote, and where I just used typing-full...)